src/components/GenderSelector.module.scss

```diff
@@ -0,0 +1,53 @@
+.gender__wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-width: 800px;
+  margin: 10px; 
+
+  .label {
+    height: 70px;
+    margin-bottom: 20px;
+
+    .radio__title {
+      font-size: 1em;
+      font-weight: 700;
+      margin-bottom: 5px;
+    }
+
+    .radio__wrapper {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+
+      .radio__items {
+        display: flex;
+
+        .radio__item {
+          display: flex;
+          align-items: center;
+          margin-right: 20px;
+        }
+
+        .radio__inner {
+          margin-right: 10px;
+        }
+
+        .radio__text {
+          font-size: 1.2em;
+          font-weight: 700;
+        }
```

Comment:
if you use BEM - not include one selector into another - use one level of nesting:

```css
.radio__title {}
.radio__items {}
.radio__item {}
.radio__inner {}
.radio__text {}
```

Reply:
At least - not use it with the BEM methodology

Reply:
done